docs/getting-started/installation.rst

```diff
@@ -24,6 +24,20 @@ This guide assumes the following:
    they change the encoding on the file and prevent it from loading correctly.
 
 
+Python Virtual Environment
+-------------------------------------
+
+A virtual environment is a way to install python packages in a different location to avoid potential version conflicts with other software like MAD or RocketMAD. It’s like a standalone version of python, independent of your “normal” python. Install it with:
+
+``apt install python-virtualenv``
```

Comment:
this probably needs to include `sudo`? I think it might be better to avoid specifics and just point to the docs in case it changes (https://virtualenv.pypa.io/en/latest/user_guide.html)